src/checker.rs

```diff
@@ -174,6 +200,52 @@ impl Checker for CppCode {
         AC.is_match(code)
     }
 
+    fn is_func_space(node: &Node) -> bool {
+        matches!(
+            node.object().kind_id().into(),
+            Cpp::TranslationUnit
+                | Cpp::FunctionDefinition
+                | Cpp::FunctionDefinition2
+                | Cpp::FunctionDefinition3
+                | Cpp::FunctionDefinition4
```

Comment:
This was not present before and is present now, so it is a behavior change. Could you add a test to cover it?